### DISCUSSION TITLE
Help with deploying to Heroku

### DISCUSSION THREAD

**Discussion by wubbzp** [QuiteAFancyEmerald/InvisiProxy#259] [General]
When I try to deploy my proxy to Heroku it gives me this error: 

**(Name of app) could not be created:
We couldn't deploy your app because the source code violates the Salesforce Acceptable Use and External-Facing Services Policy.**

What do I do? I tried Azure and Glitch but couldn't get them to work. I do own a few domains that I could use but I couldn't understand how to do it. I did look at the "readme.md".

**Comment by nonoguynono** [comment]
what is wrong with glitch? i use it to deploy all the time. if it is not loading when you deploy, you need to wait a wile. if that still does not load it, then close it out and open the new project you should now have in your home page. then it should open ,the project page. if you want a custom domain read the readme.md and read [adding-a-custom-domain](https://glitch.happyfox.com/kb/article/9-adding-a-custom-domain/). if you just want a (project name).glitch.me domain then once you open to the project page, click settings and then "edit details on project page". another way to get to the project page is to go to glitch.com/~(project name). once on the project page dashboard click edit details, and then change the project name. once the project name is changed, you can now go to the new link to your deployment of holy unblocker. the link will be (project name).glitch.me

**Reply by wubbzp** [reply]
well, glitch recently got blocked as in the glitch.me domain but not the glitch.com domain.
